README.md

```diff
@@ -27,6 +27,10 @@ etcd is used [in production by many companies](./ADOPTERS.md), and the developme
 
 See [etcdctl][etcdctl] for a simple command line client.
 
+![etcd reliability is important](logos/etcd-xkcd-2347.png)
+
+Credits to Josh Berkus and xkcd.com/2347
```

Comment:
We could use [subscript](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text) here for the footnote, might be a good idea to separate credit for creation vs alteration also?

```suggestion
<sub>Original image credited to  xkcd.com/2347, alterations by Josh Berkus.</sub>
```

Reply:
Awesome suggestion!